src/parameterizations/lateral/MOM_thickness_diffuse.F90

```diff
@@ -437,6 +452,18 @@ subroutine thickness_diffuse(h, uhtr, vhtr, tv, dt, G, GV, US, MEKE, VarMix, CDp
     endif
   endif
 
+  if (use_VarMix) then
+    if (use_gradient_model) then      !< Gradient model (Khani & Dawson, JAMES 2023)
+      if (CS%grad_Khani_scale > 0.0) then
+      !$OMP do
+        do k=1,nz ; do J=js-1,je ; do i=is,ie
+          KH_v(i,J,k) = 1.0 * CS%grad_Khani_scale * VarMix%L2grad_v(i,J) * VarMix%VH_grad(i,J,k)
+          !! print*, "KH_v =", KH_v(i,J,k)
```

Comment:
Please remove this commented out print statement.  Also, why is there the multiplication by 1.0 in the line above?